docs/modules/ROOT/pages/how-to/vshn-example/keycloak-brokering.adoc

```diff
@@ -140,3 +140,25 @@ Left empty for the default configuration for VSHN.
 The "Ignore entries pattern" takes precedence over this pattern.
 <3> These fields work the same as the fields with the same names for the "Claim To Group" mapper.
 Make sure these fields are configured the same for the "Claim To Group" and "Claim To Attribute" mappers if you want to match attribute values to group names.
+
+
+== (Optional) Configure as Default Identity Provider
+
+You have the option to set Keycloak as the default identity provider, which will make the {idp} automatically redirect to Keycloak.
+This can be helpful if you only have a single identity provider and no local users.
```

Comment:
There are at least 2 Keycloaks in the setup here, we should be more precise which one we mean :)
```suggestion
You have the option to set the brokered Keycloak as the default identity provider, which will make the {idp} automatically redirect to brokered Keycloak.
This can be helpful if you only have a single identity provider and no local users.
```

Reply:
Good point :)